src/base/rss/rssdownloadrule.h

```diff
@@ -88,6 +88,9 @@ namespace Rss
         bool operator==(const DownloadRule &other) const;
 
     private:
+        bool matches(const QString &articleTitle, QString expression) const;
+
+    private:
```

Comment:
Pardon, I was talking about the `private` word, there is already one, why you need a second?